### PR TITLE
Okazaki - Diagnosis - P1

### PR DESCRIPTION
**Pull request by wwysoc2**

### CODE REVIEW

**Review by clsackett — APPROVED** [review]

**Review by ebusygina — APPROVED** [review]
looks good

File: src/gdcdictionary/schemas/diagnosis.yaml [hunk 1]
```diff
@@ -18249,7 +18251,15 @@ properties:
...
         - Brain, Ventricle, NOS
+        - Breast, Left Lower Inner
+        - Breast, Left Lower Outer
+        - Breast, Left Upper Inner
+        - Breast, Left Upper Outer
         - Breast, NOS
+        - Breast, Right Lower Inner
+        - Breast, Right Lower Outer
+        - Breast, Right Upper Inner
+        - Breast, Right Upper Outer
```
Comment (wteouchicago) [inline src/gdcdictionary/schemas/diagnosis.yaml:18262]: just noting the capitalization here is different from the ticket (but i think the one here is correct)

Comment (bullenc) [reply src/gdcdictionary/schemas/diagnosis.yaml:18262]: Updated ticket to reflect correct case

**Review by wteouchicago — APPROVED** [review]

**Review by stilesj-uchicago — APPROVED** [review]
PHASE 1 changes look ok to me.

**Pull request merged**